task.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+# создаём каталог task с вложенными директориями
+mkdir -p task
```

Comment:
Можно сделать в одну команду
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```

Reply:
Да, я перва сделал в одну, потом потерял папку одну и сделал построчно, спасибо!